src/test/scala/edu/ie3/util/scala/collection/mutable/PriorityMultiQueueSpec.scala

```diff
@@ -0,0 +1,170 @@
+/*
```

Comment:
Can we delete the google.guava dependency after throwing out the tree multi maps?

Reply:
Yes we can: ff55b2d8